doc/source/development/python_devel.rst

```diff
@@ -1,17 +1,40 @@
-===========
-Development
-===========
+=============
+Getting setup
+=============
 
+Here we'll cover getting your development environment setup for contributing to OpenGHG.
 The source code for OpenGHG is available on `GitHub <https://github.com/openghg/openghg>`__.
 
 Setting up your computer
 =========================
 
-OpenGHG requires Python >= 3.8, so please install this before continuing
+You'll need `git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_ and Python >= 3.8, so please make sure you have both installed before continuing
 further.
 
-Virtual environment
---------------------
+
+Clone OpenGHG
+-------------
+
+First we'll clone the repository and make sure we're on the ``devel`` branch. This makes sure we're on the most up to date version of OpenGHG.
+
+.. code-block:: bash
+
+   git clone https://github.com/openghg/openghg.git
+   cd openghg
+   git checkout devel
+
+Next we'll get a virtual environment setup using either.
```

Comment:
Was this meant to say "using either... `pip` or `conda`?

Reply:
Just thinking it may be worth signalling more clear that it's either the `pip` section below OR the `conda` section but not both.

Reply:
Fixed.